source/account/routes.php

```diff
@@ -1,19 +1,9 @@
 <?php
 
 router('/', 'login');
-
 router('/login', 'login');
 
-router('/dummyUser', function () {
-    list($username, $password, $email) = array_values(config('dummyUser'));
-    if (createUser($username, $password, $email)) {
-        echo sprintf('User %s created with password %s',$username,$password);
-
-        return;
-    }
-
-    echo "Failed to create user";
-});
+router('/register', 'registration');
```

Comment:
The function name beginns with upper "R" this will work only on windows since , windows is not case sensetive, please change